x-pack/docs/en/rest-api/ml/update-filter.asciidoc

```diff
@@ -0,0 +1,69 @@
+[role="xpack"]
+[[ml-update-filter]]
+=== Update Filter API
+++++
+<titleabbrev>Update Filter</titleabbrev>
+++++
+
+Posts scheduled events in a calendar.
```

Comment:
I think this description should be something like "Updates properties of a filter".